public/study-group-detail.html

```
@@ -0,0 +1,179 @@
+    <section id="content" class="hidden grid grid-cols-1 lg:grid-cols-3 gap-6">
+      <div class="lg:col-span-2 border border-gray-200 dark:border-gray-700 rounded-xl p-5 bg-white dark:bg-gray-900">
+        <h1 id="name" class="text-2xl font-bold mb-2"></h1>
+        <p id="meta" class="text-sm text-gray-600 dark:text-gray-300 mb-4"></p>
+        <p id="desc" class="text-gray-700 dark:text-gray-200 mb-4"></p>
+        <div id="actions" class="flex flex-wrap gap-2 mb-5"></div>
+
+        <div id="invite-wrap" class="hidden border border-gray-200 dark:border-gray-700 rounded-lg p-4">
+          <h2 class="font-semibold mb-2">Invite user</h2>
+          <div class="flex gap-2">
+            <input id="invite-username" placeholder="Username or email" class="flex-1 rounded-lg border border-gray-300 dark:border-gray-700 bg-white dark:bg-black px-3 py-2" />
+            <button id="invite-btn" class="px-3 py-2 rounded-lg bg-teal-600 text-white hover:bg-teal-700">Invite</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)